src/engine.c

```diff
@@ -8860,26 +8868,10 @@ int good_chosen(game *g, int who, int c_idx, int o_idx,
 		vps = o_ptr->value * times * vp_mult;
 	}
 
-	/* Check for card award */
-	if (o_ptr->code & P4_GET_CARD)
-	{
-		/* Remember cards */
-		cards += o_ptr->value * times;
-	}
-
-	/* XXX Check for multiple card award */
-	if (o_ptr->code & P4_GET_2_CARD)
-	{
-		/* Remember cards */
-		cards += o_ptr->value * 2 * times;
-	}
-
-	/* XXX Check for multiple card award */
-	if (o_ptr->code & P4_GET_3_CARD)
-	{
-		/* Remember cards */
-		cards += o_ptr->value * 3 * times;
-	}
+	/* Determine amount of card award
+	 * Note: the card award is not affected by the power value
+	 */
+	cards = count_card_reward(o_ptr);
```

Comment:
No, I have verified that removing the "times" factor is a bug. It is now patched.